kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/symbol/kotlin/KSTypeImpl.kt

```diff
@@ -29,18 +29,22 @@ import org.jetbrains.kotlin.analysis.api.types.KtFunctionalType
 import org.jetbrains.kotlin.analysis.api.types.KtNonErrorClassType
 import org.jetbrains.kotlin.analysis.api.types.KtType
 import org.jetbrains.kotlin.analysis.api.types.KtTypeNullability
+import org.jetbrains.kotlin.utils.addToStdlib.safeAs
 
-class KSTypeImpl private constructor(private val type: KtType) : KSType {
+class KSTypeImpl private constructor(val type: KtType) : KSType {
```

Comment:
Isn't `internal` good enough? I'm afraid it's okay to expose element from Analysis API.